src/methods/malApi/anime/index.ts

```diff
@@ -35,6 +37,7 @@ export class MalAnime {
     if (fields != null) config.params.fields = fields.toString();
     if (limit != null) config.params.limit = limit;
     if (offset != null) config.params.offset = offset;
+    config.params.nsfw = nsfw;
```

Comment:
Would be good to do null/undefined check, or include a default value in case `nsfw` arg is null or undefined.